README.md

```diff
@@ -6,35 +6,35 @@ This solution starter was created by technologists from IBM.
 
 # Authors
 
-* **Alexandria Leggett**  Product Offering Manager & Creative Director
-* **Andrea Lucas**  Product Offering Manager & Technical Director
-* **Yolanda Rabun** _Lead Generalist_
-* **Denise Knorr** _Operations Director_
-* **Evelyn R. Anderson** _Sr. Security & IP Advisor_
-* **Sebastian Huynh** _Lead Designer_
-* **David Nixon** _Developer_ 
-* **Gerald Mitchell** _Developer_
-* **Syd Bailey** _Developer_
-* **Ann Umberhocker** _Developer_
-* **Shrey Anand** _Developer_
-* **Alexandra Devine** Designer & IP Strategist 
-* **Dylan Zucker** _Data Scientist_
-* **Kristen Ray** _Data Scientist_ 
-* **Chris Stefano** _Generalist_
-* **Ricky Ellison** _Generalist_
-* **Mike Boone** _Sr. Security & IP Advisor_
-
+- **Alexandria Leggett** Product Offering Manager & Creative Director
+- **Andrea Lucas** Product Offering Manager & Technical Director
+- **Yolanda Rabun** _Lead Generalist_
+- **Denise Knorr** _Operations Director_
+- **Evelyn R. Anderson** _Sr. Security & IP Advisor_
+- **Sebastian Huynh** _Lead Designer_
+- **David Nixon** _Developer_
+- **Gerald Mitchell** _Developer_
+- **Syd Bailey** _Developer_
+- **Ann Umberhocker** _Developer_
+- **Shrey Anand** _Developer_
+- **Alexandra Devine** Designer & IP Strategist
+- **Dylan Zucker** _Data Scientist_
+- **Kristen Ray** _Data Scientist_
+- **Chris Stefano** _Generalist_
+- **Ricky Ellison** _Generalist_
+- **Mike Boone** _Sr. Security & IP Advisor_
 
 # What is the problem?
 
-For centuries, voters in disparate communities, particularly Black voters, have experienced obstacles in casting their votes be it situational or intentionally constructed. Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, inability to access the correct voting location, and lack of information. As a result millions of minority votes go uncounted, purged, or simply not submitted do to lack of faith and trust in the system.
+American History has shown that voter suppression has led to corruption and a weak democracy that does not reflect the will of the people.Voting has been perceived an intangible and unfair process based on restrictive local processes, changing requirements, regulations, an inability to access the correct voting location, and a lackof information. As a result, millions of minorities, especially Black people, have had their votes go uncounted, purged, or simply not submitted due to lack of faith and trust in the system.
```

Comment:
- History -> history
- lackof -> lack of
- people.Voting -> people. Voting 
- (suggestion for simplicity) submitted due to lack of [faith and trust] in the system -> submitted due to a lack of trust in the system.